tee/test/generictest.go

```diff
@@ -159,17 +159,22 @@ func GenericEnclaveTest(t *testing.T, enc tee.Enclave) {
 		doWith(bp, alice.Exit, bob.Exit)
 	}
 
+	seal("exitPhase", 1)
+	bpsExit, err := enc.BalanceProofs()
+	requiree.NoError(err)
+	requiree.Len(bpsExit, 2, "users should still be in the system (frozen)")
```

Comment:
"frozen" is misleading because we use this term in the case that the operator goes offline and the contract freezes. Do you mean something like "exit locked" here?

Reply:
yes, good catch